src/renderer/src/design/icons/IconHome.tsx

```
@@ -0,0 +1,36 @@
+import type { SVGProps } from 'react'
+import { color } from '@renderer/design/styles'
+
+type Props = SVGProps<SVGSVGElement> & {
+  active?: boolean
+}
+
+const IconHome = ({ active = false, ...props }: Props) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)